misc/fluentd/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine@sha256:58e1a1bb75db1b5a24a462dd5e2915277ea06438c3f105138f97eb53149673c4
```

Comment:
Any reason not to use [`fluentd/fluentd`](https://hub.docker.com/r/fluent/fluentd/) instead?